### PR TITLE
[3100] Showcase block for homepage - FE refinements following internal design feedback and review

### PR DESCRIPTION
**Pull request by janehughes**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1435123100)

### Description of Changes Made

- Updates the new showcase block for the homepage following internal review

### How to Test

Add new showcase block with icons to the homepage and review both desktop & mobile versions

### Screenshots

<details>
  <summary>Expand to see more</summary>
tbx-large-showcase-homepage

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x] Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (https://docs.google.com/document/d/1PAWccdQ4tfaZsrEWmpDhvP3GH5RRmBOARFVp4b-kje8/edit?usp=sharing)
- [x] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [ ] Automated WCAG 2.1 tests pass
- [ ] HTML validation passes
- [ ] Manual WCAG 2.1 tests completed
- [ ] I have tested in a screen reader
- [ ] I have tested in high-contrast mode
- [ ] Any animations removed for prefers-reduced-motion
- [x] Not required

#### Sustainability

- [ ] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [ ] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [x] Not required

#### Pattern library

- [ ] The pattern library component for this template displays correctly, and does not break parent templates
- [ ] The styleguide is updated if relevant
- [x] Changes are not relevant the pattern library

### CODE REVIEW

File: tbx/static_src/sass/config/_mixins.scss [hunk 1]
```diff
@@ -136,7 +136,7 @@
     $color: var(--color--text),
     $interaction-color: var(--color--text),
     $shadow: false,
-    $offset: 2px
+    $offset: 4px
```
Comment (helenb) [inline tbx/static_src/sass/config/_mixins.scss:139]: Do we definitely want to change the default offset for all links here? I thought it was only the larger ones of 8px that Ben wanted to change.

**Review by helenb — CHANGES_REQUESTED** [review]
Thanks @janehughes this is looking good, but there are few tweaks needed in the code and a query on the underline offset changes

File: tbx/static_src/sass/components/_showcase.scss [hunk 2]
```diff
@@ -5,13 +5,15 @@
     $root: &;
 
     &__title {
+        @include font-size(heading-three);
         margin-bottom: $spacer-medium;
 
         @include media-query(large) {
+            @include font-size(heading-two);
```
[the remark this answers is not in the document]
Comment (janehughes) [reply tbx/static_src/sass/components/_showcase.scss]: Yes this one's slightly awkward. As per your suggestion, I've added a new heading variable `two-c` and a new motif class (two-c) to match. There isn't a `two-b` motif class which could be confusing, but I've added a comment explaining the naming.

File: tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/showcase_block.html [hunk 3]
```diff
@@ -35,7 +35,7 @@
                     {% if is_homepage_showcase %}
                         <h3 class="showcase__heading showcase__heading--with-tail">
                             {# The heading and arrow need to be on the same line with no whitespace to prevent the arrow being orphaned on a new line #}
-                            <span class="showcase__title-text">{{ item.heading }}</span><span class="showcase__tail">&nbsp;{% include "patterns/atoms/icons/icon.html" with name="arrow-wide" classname="showcase__arrow showcase__arrow--tail" %}</span>
+                            <span class="showcase__title-text">{{ item.heading }}</span><span class="showcase__tail">&nbsp;{% include "patterns/atoms/icons/icon.html" with name="arrow-wide" classname="showcase__arrow showcase__arrow--tail listing__arrow" %}</span>
```
[the remark this answers is not in the document]
Comment (janehughes) [reply tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/showcase_block.html]: That's fair, thanks for flagging. I've updated now.

File: tbx/static_src/sass/config/_mixins.scss (hunk 1 above)
Comment (janehughes) [reply tbx/static_src/sass/config/_mixins.scss]: My mistake on this one - reverted now

File: tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/showcase_block.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {% load wagtailcore_tags %}
 
-<div class="showcase grid__showcase {% if is_homepage_showcase %}showcase--home{% endif %}">
-    {% include "patterns/atoms/motif-heading/motif-heading.html" with heading=value.title heading_level=2 classes="showcase__title motif-heading--two motif-heading--static motif-heading--half-width" %}
+<div class="showcase grid__showcase {% if is_homepage_showcase %}showcase--home{% endif %} {% if value.intro %}showcase--with-intro{% endif %}">
+    {% include "patterns/atoms/motif-heading/motif-heading.html" with heading=value.title heading_level=2 classes="showcase__title motif-heading--two-b motif-heading--static motif-heading--half-width" %}
```
[the remark this answers is not in the document]
Comment (janehughes) [reply tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/showcase_block.html]: Oh I'm glad you flagged this as I'd updated this class name to `motif-heading--two-c` (to match the h2 heading class) but I hadn't updated the template so the drop cap wasn't displaying correctly. I've changed this now so it's looking better.

**Pull request merged**